src/main/java/log4j.properties

```diff
@@ -0,0 +1,17 @@
+log4j.rootLogger=debug, stdout, R
```

Comment:
This file should go into src/main/resources instead of src/main/java (which should only contain java source code).